R/table.R

```diff
@@ -170,7 +170,9 @@ table_fill <- function(cells, trim = TRUE) {
     }
 
     rowspan <- as.integer(html_attr(row, "rowspan", default = "1"))
+    rowspan[is.na(rowspan)] <- 1
```

Comment:
And maybe change `default` to `NA` just so the default is mentioned only in one place? Otherwise looks good, and just needs a bullet to the top of `NEWS.md`? It should briefly [describe the change](http://style.tidyverse.org/news.html) and end with `(@yourname, #issuenumber)`.

Reply:
Great idea and made the changes. The default value of `html_attr` is actually `NA_character_` but specifying it explicitly makes the code more readable, I find. 
Also added a bullet to `NEWS.md`.